### DISCUSSION TITLE
Toggle for displaying different eye materials

### DISCUSSION THREAD

**Discussion by zrksyd** [ScanMountGoat/ssbh_editor#139] [Ideas]
A toggle for displaying Final Smash / parry eyes, WoL light eyes, and WoL dark eyes. Should probably be in the Render Settings next to the Transition Material and Transition Factor. Should also include EyeL1, EyeR1, EyeL2, and EyeR2. (I actually don't know how far up those go but for the Remove Unused Materials function, that's all I went up to).

**Comment by ScanMountGoat** [comment]
Rendering the final smash materials would be a neat feature, but I'm not sure how easy it would be to implement. There are also additional numatb files like light_model.numatb and metamon_model.numatb that I haven't looked into yet. The main issue is that ssbh_wgpu is built to render the common files like model.numdlb and model.numatb. 

It looks like the light and dark numatb files use the same material names as the model.numatb like EyeR and EyeL. Are the EyeL1 and EyeR1 assigned in the model.numdlb? Swapping numatb files using ssbh_wgpu would be pretty easy. Messing with material assignments would be more involved to get working.

**Reply by zrksyd** [reply]
EyeL1 and EyeR1 are assigned in the model.numdlb and can act like an additional way to have a Final Smash / parry specific mesh show up.

**Comment by zrksyd** [comment]
Maybe do Final Smash eyes as an initial implementation and then get around to the other NUMATB's at a later time?
